### PR TITLE
Lazy wire up datetime

### PR DESCRIPTION
**Pull request by ehebert**
Build upon recently added `get_last_traded_dt` so that following algocode works again.

```
def handle_data(context, data):
    data[sid(24)].dt
    data[sid(24)].datetime
```

### CODE REVIEW

File: zipline/data/us_equity_pricing.py [hunk 1]
```diff
@@ -481,6 +481,13 @@ def _spot_col(self, colname):
             col = self._spot_cols[colname] = self._table[colname]
         return col
 
+    def get_last_traded_dt(self, asset, day):
+        volumes = self._spot_col('volume')
+        ix = self.sid_day_index(asset, day)
+        while True:
+            if volumes[ix] != 0:
```
Comment (jbredeche) [inline zipline/data/us_equity_pricing.py]: shouldn't we be decrementing `ix` in this loop?

Comment (ehebert) [inline zipline/data/us_equity_pricing.py]: Yes, we should. Adding a test to cover daily in the  test_dataportal suite which includes an empty day.

Comment (ehebert) [inline zipline/data/us_equity_pricing.py]: @jbredeche https://github.com/quantopian/zipline/commit/42b95c60eeec8282b78804212d5abea18948942e should fix this, have time to take a look?

**Comment by jbredeche** [comment]
lgtm
